### PR TITLE
Adapter to latest sprotty, add mandatory code

### PR DESCRIPTION
**Pull request by huyuwen**

### CODE REVIEW

File: theia-yang-extension/src/frontend/yangdiagram/yang-diagram-server.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { inject, injectable } from "inversify"
+import { LSTheiaDiagramServer } from "sprotty-theia/lib"
+import { Action, ActionHandlerRegistry, IModelFactory, TYPES } from "sprotty";
+
+@injectable()
+export class YangDiagramServer extends LSTheiaDiagramServer {
```
Comment (JanKoehnlein) [inline theia-yang-extension/src/frontend/yangdiagram/yang-diagram-server.ts]: This entire class should not be neccessary

File: theia-yang-extension/src/frontend/yangdiagram/theme-manager.ts [hunk 2]
```diff
@@ -2,8 +2,8 @@ import { Disposable } from "@theia/languages/lib/browser";
 import { ThemeService, Theme } from "@theia/core/lib/browser/theming";
 import { injectable } from "inversify";
 
-const darkTheme = require('yang-sprotty/css/dark/dark.useable.css')
-const lightTheme = require('yang-sprotty/css/light/light.useable.css')
+const darkTheme = require('yang-sprotty/css/dark/diagram.css')
```
Comment (JanKoehnlein) [inline theia-yang-extension/src/frontend/yangdiagram/theme-manager.ts]: Did the `*.useable.css` files cause any problems?

**Comment by JanKoehnlein** [comment]
Thanks a lot for the updates!

**Comment by kittaakos** [comment]
@huyuwen, was the LSP-based semantic highlighting removed on purpose or accidentally?
See here: https://github.com/theia-ide/yangster/pull/77/files#diff-459b4a152167fb95965416e04e5c9827L56-L60

**Comment by huyuwen** [comment]
Hi Akos,

I think that should be by accidentally during merge bug fix or something.

Akos Kitta <notifications@github.com> 于2019年8月16日周五 下午1:39写道：

**Comment by kittaakos** [comment]
> Hi Akos, I think that should be by accidentally during merge bug fix or something.

Thank you! I will take care of re-adding it.

**Pull request merged**